docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -8,6 +8,14 @@ protocol.
 
 When the files are changed on the filesystem, Envoy will automatically update its configuration.
 
+.. note::
+
+   Envoy only updates when the configuration file is replaced, and not when the file is edited in place.
+
+   It is implemented this way to ensure configuration consistency.
+
+   Please see the section on :ref:`atomic swaps <config_runtime_symbolic_link_swap>` for further information on how runtime configuration is updated.
```

Comment:
lets move this back to the the sealso section

Reply:
Done.